Bottom: ada6090683bcc949fbe723511a5ab5aabdc3b96d
Top:    f1237acd8dc663f5474a40d0e19a4bf8c8a3f083
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-11 15:30:38 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index b2add35..ff94ccf 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
